src/app/cases/viewer/case-viewer.component.ts

```diff
@@ -43,13 +44,13 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
     // Clone and sort tabs array
     this.sortedTabs = this.orderService.sort(this.caseDetails.tabs);
 
-    // Clone and sort fields array
-    this.sortedTabs = this.sortedTabs.map(tab => Object.assign({}, tab, {
-      fields: this.orderService.sort(tab.fields)
-    }));
-
     this.caseFields = this.getTabFields();
 
+    // Clone and sort fields array
+    this.sortedTabs = this.sortedTabs
+      .map(tab => Object.assign({}, tab, { fields: this.orderService.sort(tab.fields) }))
```

Comment:
We have  a field utils class where we are adding utility method for fields. Things like cloning ecc. Can we reuse the clone method in there or move this 'cloning' method there?

Reply:
Field utils generally has methods for merging caseFields and formFields, but we're having a two level data being merged & sorted for caseTabs and caseTabsFields, so I prefer this one to stay here